web/src/main/kotlin/no/nav/su/se/bakover/web/routes/søknad/SøknadRoutes.kt

```diff
@@ -48,27 +53,69 @@ internal fun Route.søknadRoutes(
             )
         }
     }
+
+    authorize(Brukerrolle.Saksbehandler) {
+        post("$søknadPath/{søknadId}/lukk") {
+            call.withSøknadId { søknadId ->
+                Either.catch { deserialize<Søknad.LukketSøknadBody>(call) }.fold(
```

Comment:
Det var tydeligvis her jeg skulle startet PRen! Web-laget eier alle json-typene som går til/fra frontend og bør ligge her. LukketSøknadBody kan fint flyttes inn i SøknadRoutes eller legges ved siden av. Jeg foreslår også å kalle den LukkSøknadJson.

Jeg foreslo jo først å ha et endepunkt per kommando, men hvis frontend synes det er lettere å kalle et felles endepunkt for å lukke søknad, så er det fint. Men da er det LukkSøknadJson sammen med denne metoden sitt ansvar og konvertere LukkSøknadJson til en domene-type, f.eks. LukkSøknad eller LukkSøknadCommand, evt. bare sende inn parametrene, hvis det er 2 eller færre.

Evt., så kan man i web-laget kjøre en when(lukketSøknadJson.type) og kalle respektive metoder i service-laget.